tools/ccm-service-kit/composer/dags/dependencies/queries/ccm_queries.py

```diff
@@ -0,0 +1,451 @@
+            SELECT
+            2 AS cloud_provider_id,
+            invoice_id AS invoice_id,
+            billing_account_id AS billing_account_id,
+            CAST(billing_period_start_date AS TIMESTAMP) AS billing_start_date,
+            CAST(billing_period_end_date AS TIMESTAMP) AS billing_end_date,
+            CAST(service_period_start_date AS TIMESTAMP) AS usage_start_date,
+            CAST(service_period_end_date AS TIMESTAMP) AS usage_end_date,
+            resource_location AS resource_location_id,
+            meter_region AS region_id,
+            invoice_section_id AS project_id,
+            invoice_section_name AS project_name,
+            meter_id AS service_id,
+            product_id AS charge_id,
+            product_name AS charge_description,
+            product_name AS product_name,
+            CAST(cost_in_usd AS FLOAT64) AS cost,
+            billing_currency AS currency,
+            charge_type AS charge_type,
+            quantity AS usage_quantity,
+            unit_of_measure AS usage_unit_of_measure,
+            ifnull(service_category,
+                'other'),
+            ifnull(service_type,
+                'other'),
+            current_date AS loading_date
+            FROM
+            `{project_id}.{dataset_staging}.azure_billing` AS row_data
+            LEFT OUTER JOIN (
+                SELECT
+                  *
+                FROM
+                    `{project_id}.{dataset_staging}.service_compare` 
+                WHERE
+                    cloud_provider = 'Azure') AS compare
+            ON
+            row_data.product_name = compare.product;"""
+
+
+
+    AWS_INSERT = f""" INSERT INTO
+            `{project_id}.{dataset_staging}.unified_list`
+            SELECT
+            3 AS cloud_provider_id,
+            invoice_id AS invoice_id,
+            CAST(payer_account_id AS STRING) AS billing_account_id,
+            CAST(usage_start_date AS TIMESTAMP) AS billing_start_date,
+            CAST(usage_end_date AS TIMESTAMP) AS billing_end_date,
+            CAST(usage_start_date AS TIMESTAMP) AS usage_start_date,
+            CAST(usage_end_date AS TIMESTAMP) AS usage_end_date,
+            availability_zone AS resource_location_id,
+            availability_zone AS region_id,
+            null AS project_id,
+            null AS project_name,
+            resource_id AS service_id,
+            null AS charge_id,
+            item_description AS charge_description,
+            product_name AS product_name,
+            CAST(cost AS FLOAT64) AS cost,
+            'USD' AS currency,
+            usage_type AS charge_type,
+            usage_quantity AS usage_quantity,
+            null AS usage_unit_of_measure,
+            ifnull(service_category,
+                'other'),
+            ifnull(service_type,
+                'other'),
+            current_date AS loading_date
+            FROM
+            `ccm-dev-sandbox.cloud_cost_staging.aws_billing` AS row_data
```

Comment:
Seems like this is still pointing to your dev env. If so, this should be replaced with env variables or with a placeholder and the user should be instructed to replace them. 

Also these tables are created how? Should the user create them manually, are they created in terraform?